### DISCUSSION TITLE
Using survey plugin in version 6.3.1

### DISCUSSION THREAD

**Discussion by fspinar** [jspsych/jsPsych#2753] [Q&A]
Hi all,

I need to show a survey with multiple pages, all using multi-choice questions. But I need to make them so that there is one question per page, and it is possible to go back and forth between pages. I think this is possible with the survey plugin, but I am using version 6.3.1 and I don't that plugin installed in this version. Is there a way to handle this without changing versions?

Thanks!

**Comment by jodeleeuw** [comment]
Hi @fspinar,

Unfortunately the plugin is not backwards compatible. You'll probably need to switch to version 7 to achieve what you are after.

**Comment by fspinar** [comment]
Hmm I see. Thanks @jodeleeuw 
I have another quick question. Can I add images to the survey questions in any of the survey plugins of version 6.3.1 ?
Also I might need to customize locations of questions and response buttons.

Thanks a lot!

**Reply by jodeleeuw** [reply]
You might find the [survey html form plugin](https://www.jspsych.org/7.3/plugins/survey-html-form/) to be the most useful option then. It is the most customizable option for designing survey questions, but requires more coding to get things to display as you want.
